app/src/main/java/xyz/block/gosling/features/agent/Tools.kt

```diff
@@ -217,16 +219,19 @@ object MobileMCP {
 
 }
 
+private const val appLoadTimeWait: Long = 2500
+private const val cordinateHint =
+    "(coordinates are of form: [x-coordinate of the left edge, y-coordinate of the top edge, " +
+            "x-coordinate of the right edge, y-coordinate of the bottom edge])"
```

Comment:
I think I've seen the agent open google search twice because the first time it didn't think it was open. Does this fix that? If so, awesome!